### PR TITLE
fix issues introduced in #3445 with ModularForm/GL2/Q/holomorphic/2278/2/c/c/

### PR DESCRIPTION
**Pull request by alexjbest**
Sorry for the hassle my previous PR #3445  broke display a little.

Compare https://scratch.lmfdb.xyz/ModularForm/GL2/Q/holomorphic/2278/2/c/c/ with https://red.lmfdb.xyz/ModularForm/GL2/Q/holomorphic/2278/2/c/c/ the basis display was broken.

### CODE REVIEW

File: lmfdb/classical_modular_forms/web_newform.py [hunk 1]
```diff
@@ -634,9 +634,9 @@ def _make_frac(self, num, den):
                 return "%s\(/%s\)" % (num, den)
         else:
             if paren:
-                return r"\((\)%s\()/\)%s" % (num, bigint_knowl(den))
+                return r"\((\)%s\()/%s\)" % (num, make_bigint(web_latex(den, enclose=False)))
             else:
-                return r"%s\(/\)%s" % (num, bigint_knowl(den))
+                return r"%s\(/%s\(" % (num, make_bigint(web_latex(den, enclose=False)))
```
Comment (roed314) [inline lmfdb/classical_modular_forms/web_newform.py]: Shouldn't this be `\)` rather than `\(`?

File: lmfdb/utils/utilities.py [hunk 2]
```diff
@@ -684,7 +684,7 @@ def bigint_knowl(n, cutoff=20, max_width=70, sides=2):
             lng = r"\(%s\)" % lng
         return r'<a title="[bigint]" knowl="dynamic_show" kwargs="%s">\(%s\)</a>'%(lng, short)
     else:
-        return r'\(%s\)'%n
+        return r'%s'%n
```
Comment (roed314) [inline lmfdb/utils/utilities.py]: Why make this change?  Don't we still want mathmode even when not including a knowl?

**Comment by edgarcosta** [comment]
The broken changes are now live: https://beta.lmfdb.org/ModularForm/GL2/Q/holomorphic/2278/2/c/c/

**Comment by AndrewVSutherland** [comment]
@alexjbest Is there any chance you can fix the issue @roed314 noted in web_newform.py (which also looks like a typo to me).  I'd like to get this merged and pushed to beta.

**Pull request merged**